tests/testthat/test-edit-spreadsheets.R

```diff
@@ -0,0 +1,81 @@
+context("edit spreadsheets")
+
+test_that("Make a new spreadsheet", {
+  skip("Can not use login() to talk to Drive API")
```

Comment:
Seems like we're skipping these tests entirely now, am I right? We should narrow this down.